src/TTextEdit.cpp

```diff
@@ -1232,6 +1226,13 @@ void TTextEdit::copySelectionToClipboard()
     clipboard->setText(selectedText);
 }
 
+void TTextEdit::selectAll()
```

Comment:
It does not seem necessary to use a wrapper implementation for this, why not call this `slot_selectAll()` directly... 😕